controller/src/main/java/io/pravega/controller/util/Config.java

```diff
@@ -148,6 +152,7 @@
     private static final Property<Integer> PROPERTY_TXN_TTL_HOURS = Property.named("transaction.ttlHours", 24);
     private static final Property<String> PROPERTY_SCALE_STREAM_NAME = Property.named("scale.streamName", "_requeststream");
     private static final Property<String> PROPERTY_SCALE_READER_GROUP = Property.named("scale.ReaderGroup", "scaleGroup");
+    private static final Property<Boolean> PROPERTY_DUMP_STACK_ON_SHUTDOWN = Property.named("dumpStackOnShutdown", false);
```

Comment:
Question: is this matching the new way of setting configuration properties for the controller? Do we need to prefix the property with any name, like `service.`?

Reply:
this is same as process level configs for example:
```
    private static final Property<Boolean> PROPERTY_HOST_MONITORING_ENABLED = Property.named("hostMonitorEnabled", true);
```

Reply:
For organization purposes, can we group this property with the ones of the same category (process level according to you)?